include/usgscsm/UsgsAstroSarSensorModel.h

```diff
@@ -249,6 +249,7 @@ class UsgsAstroSarSensorModel : public csm::RasterGM, virtual public csm::Settab
     std::vector<double> m_sunPosition;
     std::vector<double> m_sunVelocity;
     double m_wavelength;
+    std::string m_lookDirection;
```

Comment:
I would rather look direction be an enumeration or something. Comparing a string to determine left/right is not good. It's better to have either an enumeration or maybe a boolean.

Reply:
Yeah, you're right. I'd thought of making it an enum, and I guess I just got lazy.